sdks/java/io/cdc/src/main/java/org/apache/beam/io/cdc/DebeziumOffsetTracker.java

```diff
@@ -36,10 +38,11 @@
     @Override
     public boolean tryClaim(Map<String, Object> position) {
         LOG.info("-------------- Claiming {} used to have: {}", position, restriction.offset);
-
+        DateTime currentTime = new DateTime();
+        long elapsedTime = currentTime.getMillis() - KafkaSourceConsumerFn.startTime.getMillis();
```

Comment:
You can use `System.currentTimeMillis()` instead of creating the `DateTime` object
```suggestion
        long elapsedTime = System.currentTimeMillis() - KafkaSourceConsumerFn.startTime.getMillis();
```